target/linux/ath79/dts/ar7240_openmesh_om2p-v1.dts

```diff
@@ -91,7 +91,6 @@
 		compatible = "jedec,spi-nor";
 		reg = <0>;
 		spi-max-frequency = <40000000>;
-		linux,mtd-name = "ar7240-nor0";
```

Comment:
maybe the boot arguments contain an argument to use the rootfs from `ar7240-nor0`. This is the reason why we need this patch on `pistachio`.